README.md

```diff
@@ -34,13 +34,13 @@ the `env` field, or you may face an `API rate limit exceeded` error.
       MY_FIRST_VAR=VALUE
       MY_SECOND_VAR=VALUE
 
-    # Specify an alternative configuration file for this execution.
+    # Specifies an alternative configuration file for this execution.
     # Default: .sauce/config.yml
     config-file: .sauce/myconfig.yml
 
     # Specifies the Sauce Labs data center through which tests will run.
     # Valid values are us-west-1 or eu-central-1.
-    # Default: us-west-1
+    # `region` is mandatory and should be set either in the Sauce Config file or by setting this field.
```

Comment:
Don't think this note is necessary? Probably enough to just remove the note about the old default value?

Reply:
Or at least, specify what happens if `region` is set in the config and here as part of the action's configuration.

Reply:
hmm, compared with other mandatory fields setting, I think removing it is enough.